src/promptflow/promptflow/batch/_csharp_executor_proxy.py

```diff
@@ -14,6 +14,7 @@
 
 EXECUTOR_SERVICE_DOMAIN = "http://localhost:"
 EXECUTOR_SERVICE_DLL = "Promptflow.dll"
+EXECUTOR_INIT_ERROR_FILE = "init_error.json"
```

Comment:
Is the file name a part of the contract between the csharp executor and executor proxy? (Is this file name hardcoded in the csharp side too?)

Reply:
Oh I see. It is passed via the command line. Ignore me.

Reply:
Use random file name here.